packages/teams-js/src/private/chat.ts

```diff
@@ -18,64 +18,47 @@ import { ChatMembersInformation } from './interfaces';
  */
 export namespace chat {
   /**
-   * @hidden
-   * Hide from docs
-   * --------------
-   * Allows the user to start or continue a conversation with each subentity inside the tab
    *
+   * Allows the user to start a direct chat with a user
+   * @members = length 1
```

Comment:
what is the @member tag here? I can't find it on the official Typedoc docs.

Reply:
I don't think it generates anything as is because it's @hidden. When I check out the branch and remove the @hidden tag to run `yarn docs` I get this:
<img width="834" alt="image" src="https://user-images.githubusercontent.com/69540388/158465965-46d0acf6-551e-4867-94f1-77bfddd2bc77.png">

So it won't break, but I feel like this isn't the way we want to specify required length of members. (Seeing now that members is probably just openChatRequest's members and not a Typedoc tag like @remarks, @hidden, etc.) I would expect to see something like 

```
@param openChatRequest The request to open a chat for. Length must be equal to 1.
```

And maybe including more details pending additional implementation on what happens with Trevor's other comment below on making it impossible for app devs to call this API incorrectly.